src/main/java/pages/InformationPage.java

```diff
@@ -0,0 +1,87 @@
+package pages;
+
+import common.AbstractComponent;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import model.Customer;
+
+public class InformationPage extends AbstractComponent {
+
+    WebDriver driver;
+    Select selectState;
+    Select selectCountry;
+    WebDriverWait wait;
+
+    public InformationPage(WebDriver driver){ super(driver); }
+
+    @FindBy(xpath = "//input[@id='checkout_email']")
+    private WebElement userEmail;
```

Comment:
userEmailTextbox is a better name since it describes what element exactly is it. apply to other variables too